authority-portal-frontend/src/app/pages/authority-connector-detail-page/authority-connector-detail-page/authority-connector-detail-page.component.ts

```diff
@@ -130,7 +131,7 @@ export class AuthorityConnectorDetailPageComponent
 
   copyToClipboard(s: string | undefined) {
     if (s) {
-      this.clipboard.copy(s);
+      this.clipboardUtils.copyToClipboard(s);
     }
```

Comment:
move the if into the copyToClipboard method, you could show a toast "Nothing to copy"

Reply:
done